### DISCUSSION TITLE
How do I conditionally include JavaScript in the head from a specific template file?

### DISCUSSION THREAD

**Discussion by RyanRoberts** [statamic/cms#10705] [Q&A] [answered]
What’s the best way to include some JavaScript in the head of my document (the head being in the main layout file) from a specific template file?

For example, I have an article template, and only pages using that template need this specific JavaScript. I’d rather not have it load on other pages, in fact the JavaScript might not even be needed at all if the site’s owner doesn’t use a particular block that relies on it (it's a Lightbox image popup thing).

**Comment by ryanmitchell** [comment, marked as answer]
Stacks are the way to go: https://statamic.dev/antlers#stacks

**Reply by duncanmcclean** [reply]
I was just about to say this 💯

**Reply by RyanRoberts** [reply]
Man, I learn something new and amazing with Statamic every day!
